src/coreclr/src/jit/importer.cpp

```diff
@@ -3425,7 +3425,7 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                                 CORINFO_CLASS_HANDLE    clsHnd,
                                 CORINFO_METHOD_HANDLE   method,
                                 CORINFO_SIG_INFO*       sig,
-                                unsigned                methodFlags,
+                                unsigned&               methodFlags,
```

Comment:
Can you update comment in the block above:
* "Arguments" -- `methodFlags` is now `[IN,OUT]`
* "Returns" -- `methodFlags` are modified if we end up effectively devirtualizing the call as part of intrinsic expansion